add-sitemap-to-redirects-webpack-plugin.js

```diff
@@ -0,0 +1,32 @@
+function AddSitemapToRedirectsWebpackPlugin() {
+  return {
+    apply: (compiler) => {
+      compiler.hooks.emit.tapAsync('AddSitemapToRedirectsWebpackPlugin', emit);
+    },
+  };
+}
+
+function emit(compilation, callback) {
+  const tenant = process.env.SCRIVITO_TENANT;
+  const url = process.env.DEPLOY_PRIME_URL;
+
+  if (url && tenant) {
+    const redirects = compilation.assets['_redirects'].source().toString();
+    const patchedRedirects = redirects.replace(/^\/\* \/.+$/m, rewriteRule(tenant, url));
+
+    compilation.assets['_redirects'] = {
+      source: () => patchedRedirects,
+      size: () => patchedRedirects.length,
+    };
+  }
+
+  callback();
+}
+
+function rewriteRule(tenant, url) {
```

Comment:
Some issues here:

* It's actually *two* rules
* `/_sitemaphtml /_sitemap.html 200` can be hardcoded
* The readability can be improved by having blank lines, like:
```js
return `
/rule1
/rule2
`;
```

* [ ] Try improving readability